include/git2/tree.h

```diff
@@ -381,7 +381,7 @@ GIT_EXTERN(int) git_treebuilder_write(
  *
  * @see git_treebuilder_write
  *
- * @param id Pointer to store the OID of the newly written tree
+ * @param oid Pointer to store the OID of the newly written tree
```

Comment:
This change conflicts with the one from @ethomson regarding the same issue. Only that he changed the argument's name in the function prototype instead of the docs. Hopefully we will not end up with both patches applied, resulting in the same but reversed issue :P